tools/block-generator/upload_metrics.py

```diff
@@ -0,0 +1,57 @@
+from datadog import initialize
+from datadog import api
+import os
+import argparse
+
+parser = argparse.ArgumentParser(description="Upload performance metrics to Datadog")
+parser.add_argument(
+    "-f",
+    "--perf-reports",
+    required=True,
+    action="store",
+    dest="files",
+    type=str,
+    nargs="*",
+    help="list of reports created by the block generator",
+)
+parser.add_argument(
+    "-c",
+    "--binary-version",
+    required=True,
+    help="Release version or the commit hash of the Conduit binary used during the performance test",
+)
+args = parser.parse_args()
+
+
+def parseReport(report):
+    data = dict()
+    with open(report) as f:
+        for line in f:
+            tag, value = line.split(":")
+            if tag == "scenario":
+                data[tag] = value
+            else:
+                data[tag] = float(value)
```

Comment:
```suggestion
            tag, value = line.split(":")
            data[tag] = value if tag == "scenario" else float(value)
```
ignorable suggestion to use python ternary